pkgs/racket-doc/scribblings/reference/stx-trans.scrbl

```diff
@@ -232,7 +232,7 @@ identifier, the @racket[exn:fail:contract] exception is raised.
 
 @defproc[(local-expand [stx any/c]
                        [context-v (or/c 'expression 'top-level 'module 'module-begin list?)]
-                       [stop-ids (or/c (listof identifier?) (cons/c 'only (listof identifier?)) #f)]
+                       [stop-ids (or/c (listof identifier?) empty #f)]
```

Comment:
I’d just remove the `empty` special case altogether. It is subsumed by the `(listof identifier?)` contract, and with other changes, an empty list is less special than it used to be. I think documenting it separately is just confusing.